source/_components/climate.systemair_savecair.markdown

```diff
@@ -0,0 +1,45 @@
+---
+layout: page
+title: "Systemair Savecair Climate"
+description: "Instructions on how to integrate Systemair Savecair into Home Assistant."
+date: 2018-10-19 05:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: systemair.png
+ha_category: Climate
+ha_release: 0.81
+ha_iot_class: "Cloud Polling"
+---
+The `systemair savecair` climate platform consumes data from the [Savecair web interface](https://savecair.systemair.com). This component enables you to view and configure the climate decive.
+
+To enable this platform, you need the Savecair module configured.
+
+```yaml
+# Example configuration.yaml entry
+climate:
+  - platform: systemair_savecair
+    device: vtr_300
+    name: System AIR
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.